src/server/api/routers/airtable.ts

```diff
@@ -1,32 +1,50 @@
 import Airtable from "airtable";
 
 import { createTRPCRouter, publicProcedure } from "eufs-finance/server/api/trpc";
+import { date, string } from "zod";
+
+// Define the interface for a reimbursement
+interface reimbursements{
+    price: number;
+    team: string;
+    sub_team: string;
+    date: string;
+}
+
 
 const base = new Airtable({apiKey: 'patYgRTIOPRIpKD0D.68f7c75b875a70c115c8a26b80907d60599861c00968e4d42dab2a7e87383f04'}).base('appeXw3HbMoKHSeYM');
 
 export const airtableRouter = createTRPCRouter({
   requests: publicProcedure
     .query( () => {
+        //initialise an array to store reimbursements
+        const rows: reimbursements[] = []; 
          
       base('Requests').select({
-          // Selecting the first 3 records in Table:
-          maxRecords: 3,
           view: "Table"
       }).eachPage((records, fetchNextPage) => {
           // This function (`page`) will get called for each page of records.
-
-          records.forEach(function(record) {
-              console.log('Retrieved', record.get('ID'));
-          });
+          const row = records.map(record =>({
```

Comment:
not sure about the name since these are multiple rows, so we could call this `row_subset` or something similar